src/transformers/models/fuyu/configuration_fuyu.py

```diff
@@ -206,3 +208,20 @@ def _rope_scaling_validation(self):
             )
         if rope_scaling_factor is None or not isinstance(rope_scaling_factor, float) or rope_scaling_factor <= 1.0:
             raise ValueError(f"`rope_scaling`'s factor field must be a float > 1, got {rope_scaling_factor}")
+
+    @property
+    def vocab_size(self):
+        warnings.warn(
+            "The `vocab_size` attribute is deprecated and will be removed in v4.43, Please use `text_config.vocab_size` instead.",
```

Comment:
tiny nit - we should do this over two releases 

```suggestion
            "The `vocab_size` attribute is deprecated and will be removed in v4.44, Please use `text_config.vocab_size` instead.",
```

Reply:
oke, will change for 4.44 in paligemma also